tests/framework/di/InstanceTest.php

```diff
@@ -46,7 +46,27 @@ public function testEnsure()
         $this->assertTrue(Instance::ensure(['class' => 'yii\db\Connection', 'dsn' => 'test'], 'yii\db\Connection', $container) instanceof Connection);
     }
 
-    public function testEnsureWithoutType()
+    /**
+     * ensure an InvalidConfigException is thrown when a component does not exist.
```

Comment:
perhabs in normal code, not in a test environment. Tests should be small and self declaring. Your method docs are useless here.